napari/tests/test_advanced.py

```diff
@@ -190,7 +190,7 @@ def test_changing_display_surface(qtbot):
     )
 
     assert len(viewer.layers) == 1
-    assert view.layers.vbox_layout.count() == 2 * len(viewer.layers) + 2
+    assert view.layers.vboxLayout.count() == 2 * len(viewer.layers) + 2
```

Comment:
Plus the diff is gonna be so much smaller! =)

Reply:
done ;-)